vector-im/element-android
# [Device Management] Learn more bottom sheets (PSG-715)

## Type of change


## Content


Adding a new bottom sheet to display explanations when pressing learn more links in the device manager screens.

## Motivation and context


Closes #7100 

## Screenshots / GIFs





<img src="https://user-images.githubusercontent.com/46314705/192783329-1916d857-2b19-49e2-8c11-05af6c67ffc4.gif" width=300 />

## Tests



- Enable the new device manager feature flag
- Go to Settings -> Security & privacy -> Show all Sessions (V2, WIP)
- Navigate in all the device management screens and press the different learn more texts
- Check a bottom sheet with correct text is displayed

## Tested devices

- OS version(s): Android 11

## Checklist
